frontend/src/mixins/shootItem.js

```diff
@@ -317,11 +317,12 @@ export const shootItem = {
     ...mapActions(useProjectStore, [
       'projectNameByNamespace',
     ]),
-    shootActionToolTip (tooltip) {
+    shootActionToolTip (tooltip, overrideDeletionFlag) {
```

Comment:
ok I also thought about doing that since it is only used in two places after the recent refactoring. If you also consider this to be the way to go I will do this